src/bidiMapper/domains/context/bidiPreloadScript.ts

```diff
@@ -48,7 +48,9 @@ export class BidiPreloadScript {
   /** Browsing context ID. */
   readonly #contextId: CommonDataTypes.BrowsingContext | null;
   /** Targets, in which the preload script is initialized. */
-  readonly #targetIds = new Set<string>();
+  readonly #targetIds: Set<string> = new Set<string>();
```

Comment:
I believe ESLint would remove the type annotation in this case as it can be inferred from the value.

Reply:
ok, removed